bundles/automation/org.eclipse.smarthome.automation.module.core/src/main/java/org/eclipse/smarthome/automation/module/core/handler/ItemStateTriggerHandler.java

```diff
@@ -42,6 +42,7 @@
 
     private String itemName;
     private String state;
+    private String oldState;
```

Comment:
ok, I was mislead as I thought this is about the module outputs - now I see that this is a configuration parameter.